src/main/java/seedu/duke/command/HelpCommand.java

```diff
@@ -20,16 +20,15 @@
 
     @Override
     public String execute() {
-
         return HELP_STRING + InterfaceManager.LS + COLOR_WHITE_STRING
                 + AddNoteCommand.getCommandUsageNote() + InterfaceManager.LS + COLOR_CYAN_STRING
```

Comment:
AddNoteCommand.getCommandUsageNote() change to AddEventCommand.getCommandUsage()?
```suggestion
                + AddNoteCommand.getCommandUsage() + InterfaceManager.LS + COLOR_CYAN_STRING
```